script/Ciel/Day 14/14_01A_CIEL14A_4.txt

```diff
@@ -227,7 +227,7 @@ Unbelievable...
 [sha:3bd31c6eb2ebb4a126618fea77774a9d4e3b1e88]{
 -- Page 113, Offset 37930.
 -- 人間社会との関わり。
-She had an understanding with human society.
+She had an understanding of human society.
```

Comment:
Original was correct - to have an understanding with = to have an implicit contract with